### PR TITLE
1.9 Release Notes

### PR DESCRIPTION
**Pull request by joel-hamill**
## What are your changes?
1.9 OSS Release Notes

## What is the urgency level of this change?
- [ ] Blocker
- [x] High
- [ ] Medium

## I have completed these items:
- [x] I have built locally and tested for broken links and formatting.
- [ ] If appropriate, I have added redirects.

### CODE REVIEW

File: src/releases/1.9.0/notes.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+# What's New
+
+## Apache Mesos 1.2
+
+- GPU Support
+- TaskGroups and Nested Containers (Pods)
+- Container exec/attach for debugging
+- v1 Master/Agent Operator API
+- HTTP(S) and TCP Health Checks
+- Java shim for v1 scheduler API (doesn’t work in strict mode yet) <!-- [maturity-badge status='experimental'] -->
```
Comment (pyronicide) [inline src/releases/1.9.0/notes.md]: Strict is EE.

Comment (joel-hamill) [inline src/releases/1.9.0/notes.md]: fixed

File: src/releases/1.9.0/notes.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+- Marathon scale improvements
+
+## CLI
+
+- Service account login support (useful for unattended scripts/automation)
+- New command for discovering why applications aren’t starting: `dcos marathon debug list`
+- Subcommands for package management and service management. <!-- [maturity-badge status='experimental'] -->
+- Logging commands `dcos task log` and `dcos node log` use the new [logging API](/docs/1.9/administration/logging/logging-api/) backend. This is a big improvement over using the Files API because you can now stream systemd unit logs for DC/OS system services and logs from tasks that are deployed by Mesos.
+- New debugging command: `dcos task exec`. For more information, See the debugging [documentation](/docs/1.9/administration/debugging/).
+
```
Comment (sascala) [inline src/releases/1.9.0/notes.md]: Some list items have periods at the end and some don't. Let's choose! (I like periods)

File: src/releases/1.9.0/notes.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+- Logs from Mesos tasks are tagged with appropriate metadata to allow easy searching by container ID.
+- New logging API that sits on top of journald to allow queries to these logs from the DC/OS CLI dcos node log and dcos task log commands.
+- Because all logs are going to journald, you can easily setup log aggregation solutions such as logstash to get logs into their aggregated storage solutions.
+- The new DC/OS CLI changes allow users to query Mesos task logs as well as DC/OS system service logs from one consolidated tool.
+
+For more information, see the [documentation](/docs/1.9/administration/logging/).
+
+## Metrics <!-- [maturity-badge status='experimental'] -->
+
+- Node-level HTTP API which returns metrics from frameworks, cgroup allocations per container and host level metrics such as load and memory allocation.
```
Comment (sascala) [inline src/releases/1.9.0/notes.md]: s/which/that and comma after "per container"

File: src/releases/1.9.0/notes.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+    - Pods support
+    - Support for testing against clusters with OAuth
+    - Better help output
+    - Updated documentation
+    - Utility methods for using Mesos-DNS, waiting on cluster actions and partitioning and reconnecting to DC/OS masters
+
+## Upgrades
+
+Improved upgrade tooling and experience for on-premise installations. Upgrades now use internal DC/OS API’s to ensure nodes can be upgraded with minimal disruption to running DC/OS services on a node. The upgrade procedure has also been simplified to improve user experience.
```
Comment (sascala) [inline src/releases/1.9.0/notes.md]: remove apostrophe in API's.

**Pull request merged**